packages/bits/src/lib/wizard-v2/wizard-icon.directive.ts

```diff
@@ -0,0 +1,26 @@
+import {Directive, Input, TemplateRef} from "@angular/core";
+import {StepState} from "@angular/cdk/stepper";
+
+/** Template context available to an attached `NuiWizardIcon`. */
+export interface NuiWizardIconContext {
+    index: number;
+    active: boolean;
+    optional: boolean;
+}
+
+export interface MatStepperIconContext {
+    index: number;
+    active: boolean;
+    optional: boolean;
+}
+
+@Directive({
+    selector: "ng-template[nuiWizardIcon]",
+})
+export class NuiWizardIconDirective {
```

Comment:
We typically only prefix module names and selectors with `Nui`

```suggestion
export class WizardIconDirective {
```